rsr_convert_scripts/abi_rsr.py

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 #
-# Copyright (c) 2016 - 2018 Pytroll developers
+# Copyright (c) 2016 - 2021 Pytroll developers
 #
 # Author(s):
 #
-#   Adam.Dybbroe <adam.dybbroe@smhi.se>
+#   Adam Dybbroe <Firstname.Lastname@smhi.se>
```

Comment:
To prevent spam?

Reply:
Yes, that was the intention. We should eventually add a AUTHORS.md to Pyspectral so we could remove these though.